src/main/resources/assets/api-docs.yaml

```diff
@@ -1742,6 +1742,13 @@ paths:
       summary: Manage Data Access Requests V2
       description: |
         Returns all the elections created for Data Access Requests relevant to the current user.
+      parameters:
+        - name: roleName
+          in: query
+          description: An optional roleName that the user is making the request as. A user can have multiple roles, so adding this parameter will filter the list accordingly.
```

Comment:
In swagger, we can specify a list of allowable values here. Since we're only supporting signing official, it might be good to enforce that here. See https://swagger.io/docs/specification/data-models/enums/ for an example.